sadu-queries/src/test/java/de/chojo/sadu/wrapper/mapper/Result.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *     SPDX-License-Identifier: AGPL-3.0-only
+ *
+ *     Copyright (C) 2022 RainbowDashLabs and Contributor
+ */
+
+package de.chojo.sadu.wrapper.mapper;
+
+class Result {
```

Comment:
Just use a record here.

Reply:
I lowered the target language level to 15 again.